schema.sql

```diff
@@ -1,5 +1,41 @@
 /* Database schema to keep the structure of entire database. */
 
 CREATE TABLE animals (
-    name varchar(100)
+    id SERIAL PRIMARY KEY,
+    name VARCHAR(255),
+    date_of_birth DATE,
+    escape_attempts INTEGER,
+    neutered BOOLEAN,
+    weight_kg DECIMAL(10, 2)
 );
+ALTER TABLE animals ADD COLUMN species VARCHAR(255);
+
+
+
+   CREATE TABLE species (
+   id int NOT NULL IDENTITY(1,1),
+   name varchar(255),
+   PRIMARY KEY (id)
+   );
+
+
+      CREATE TABLE species (
+   id int NOT NULL IDENTITY(1,1),
+   name varchar(255),
+   PRIMARY KEY (id)
+   );
+
```

Comment:
- [ ] Great job implementing some required changes. However, there's one table yet to be added. (i.e) `owners` table. Kindly add it and remove the duplicated `species` table.
```
CREATE TABLE owners (
   id int NOT NULL IDENTITY(1,1),
   full_name varchar(255),
   age int,
   PRIMARY KEY (id)
 );
```